pyriemann_qiskit/classification.py

```diff
@@ -305,13 +320,14 @@ class QuanticVQC(QuanticClassifierBase):
     Note there is no classical version of this algorithm.
     This will always run on a quantum computer (simulated or not)
 
-    Parameters
+    Attributes
```

Comment:
`optimizer` and `gen_var_form` are parameters since they are defined in init.
```suggestion
    Parameters
```